compiler/testSuite/11-decimal/neg1-v.bal

```diff
@@ -0,0 +1,17 @@
+import ballerina/io;
+public function main() {
```

Comment:
You need to have cases where the operand is const.

Reply:
Added test cases